CesiumGltfContent/src/GltfUtilities.cpp

```diff
@@ -971,6 +970,11 @@ void intersectRayScenePrimitive(
   if (!pPositionAccessor)
     return;
 
+  // Ignore non-float positions, these are unsupported
+  // Although valid from the glTF spec, they aren't generally useful here
+  if (pPositionAccessor->componentType != Accessor::ComponentType::FLOAT)
```

Comment:
According to the table at https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#meshes-overview, the `POSITION` accessor must use VEC3/float. But 1. it can still make sense to have this check here (and maybe even log this as an error instead of "doing nothing"), and 2. maybe there are extensions that relax this constraint (like the (pending!) extension at https://github.com/KhronosGroup/glTF/pull/2397 )

Reply:
Nice catch! I missed that part of the spec. I'll update the comments with your notes.

It could be nice to have some warnings logged somewhere when traversing a gltf with bogus values. There are other checks like this that "do nothing" and try our best when encountering bad data.

Reply:
... unless the glTF is using [KHR_mesh_quantization](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_mesh_quantization/README.md)

Reply:
Thanks, I'll add a note about that extension, and see how hard it would be to extend this PR to cover it.

Reply:
@lilleyse That was one reason why I mentioned extensions (although I might have been more explicit)
(I would have to refresh my memory about the `AccessorView` - I don't know whether this already decodes quantized data - i.e. where _exactly_ the quantization extension is handled - but being aware of that point might be enough...)

Reply:
It wasn't too bad to support quantized meshes (non-float positions). Change have been committed.